database/seedDb.js

```diff
@@ -1,38 +1,49 @@
 const faker = require('faker');
 const Image = require('./index.js');
 
-const seedDb = () => {
-  function makeOnePhoto() {
-    return {
-      url: faker.image.imageUrl(),
-      caption: faker.lorem.sentence(),
-      ownersubmitted: faker.random.boolean(),
-      rating: Math.floor((Math.random() * 5) + 1),
-      username: faker.internet.userName(),
-      thumbnail: faker.internet.avatar(),
-    };
-  }
-  function getPhotos() {
-    const photos = [];
-    const numOfPhotos = Math.floor((Math.random() * 6) + 5);
-    for (let i = 0; i < numOfPhotos; i += 1) {
-      const newPhoto = makeOnePhoto();
-      photos.push(newPhoto);
-    }
-    return photos;
+let accum = 0;
```

Comment:
Accum is used to increment the id of the image i want to use for my sample image database on the internet.